public/utils/validate.js

```diff
@@ -59,6 +59,10 @@ export const required = (value) => {
   if (!value) return 'Required.';
 };
 
+export const requiredNumber = (value) => {
+  return _.isEmpty(value?.toString()) ? 'Required.' : undefined;
```

Comment:
Should we not check whether the value is a number using something like [NaN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/NaN)?

Reply:
That makes sense. I'll revise the PR to include that.